### ISSUE TITLE
Sherlock AttributeError since upgrade

### ISSUE THREAD

**Issue by genghisken** [thespacedoctor/sherlock#35]
Hi Dave

Not sure if I'm doing something stupid, but got the following error from Sherlock since I upgraded...

```
Traceback (most recent call last):
  File "/home/atls/anaconda2/bin/sherlock", line 11, in <module>
    sys.exit(main())
  File "/home/atls/anaconda2/lib/python2.7/site-packages/sherlock/cl_utils.py", line 170, in main
    classifier.classify()
  File "/home/atls/anaconda2/lib/python2.7/site-packages/sherlock/transient_classifier.py", line 200, in classify
    self._create_tables_if_not_exist()
  File "/home/atls/anaconda2/lib/python2.7/site-packages/sherlock/transient_classifier.py", line 1300, in _create_tables_if_not_exist
    createStatement.append(trigger)
AttributeError: 'str' object has no attribute 'append'

```

**Comment by genghisken** [comment]
I changed .append to += for the trigger statement, and now I get an SQL syntax error.  However, if I try to run the actual SQL generated manually, it succeeds!  No idea what's going on...

```
.
.
DELIMITER $$
CREATE TRIGGER dateCreated
BEFORE INSERT ON `sherlock_crossmatches`
FOR EACH ROW
BEGIN
   IF NEW.dateCreated IS NULL THEN
      SET NEW.dateCreated = NOW();
      SET NEW.dateLastModified = NOW();
   END IF;
END$$
DELIMITER ; >>
The error was: (1064, u"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CREATE TABLE IF NOT EXISTS `sherlock_classifications` (\n  `transient_object_id` ' at line 77")

```

**Comment by genghisken** [comment]
So - since I already have the tables and the trigger, I commented out the ```self._create_tables_if_not_exist()``` command in transient_classifier.py.  This produces a more challenging error with multiprocessing.  See next ticket.

**Comment by thespacedoctor** [comment]
The second 'SQL syntax' issue here appears to have been caused to a [change introduced to PyMYSQL](https://github.com/PyMySQL/PyMySQL/blob/0305541132b134047eb4ba70fc9acbc9be603d05/CHANGELOG#L7-L15). I believe I have now fix this issue with 6a56bad799b59fb9e3a6df4e89b77fb18d3ee4ab and in thespacedoctor@f56ae6cc728d9cccb1ba5525a6436a246273a366 in fundamentals.
